src/styling.rs

```diff
@@ -0,0 +1,788 @@
+//! Style mathematical symbols in Unicode.
+
+use std::fmt::{self, Write};
+use std::iter::FusedIterator;
+
+/// The version of [Unicode](https://www.unicode.org/) that this version of the
+/// styling module is based on.
+pub const UNICODE_VERSION: (u8, u8, u8) = (16, 0, 0);
+
+/// A style for mathematical symbols.
+///
+/// Notation in mathematics uses a basic set of characters which can be styled.
+/// The following groupings are used in the documentation:
+/// - digits: The basic Latin digits 0–9 (U+0030..U+0039).
+/// - latin: The basic uppercase and lowercase Latin letters, a–z
```

Comment:
```suggestion
/// - Latin: The basic uppercase and lowercase Latin letters, a–z
```

"Latin" should be capitalized, same for "Greek" and "Arabic" below.